.eslintrc

```diff
@@ -14,6 +14,7 @@
     "no-useless-return": "warn",
     "no-var": "warn",
     "no-console": "warn",
+    "no-unused-vars": "off",
```

Comment:
Why disable this rule?

Reply:
It was already off before. In the original file you had:
"@typescript-eslint/no-unused-vars": "off"

However after upgrading dependencies that line has changed and should now be written as:
"no-unused-vars": "off"

(I updates typescript/eslint plugin from 2.x to 4.x)